lang/en.js

```diff
@@ -9,7 +9,6 @@ export default {
 	"components.dialog.close": "Close this dialog",
 	"components.dropdown.close": "Close",
 	"components.filter.loading": "Loading filters",
-	"components.filter.filterCount": "{number, select, max {99+} all {All} other {{number}}}",
```

Comment:
Haha too bad this didn't remove `filterCountDescription` that we had all the translation issues with.

Reply:
Yeah unfortunately we're still on the hook for those lol